test/jest.setup-file.ts

```diff
@@ -3,8 +3,9 @@
 
 
 // Mocking crypto with NodeJS WebCrypto module only for tests.
-import { webcrypto } from 'node:crypto'
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { webcrypto } = require('node:crypto');
```

Comment:
Using ESM, prefers the use of `import` instead of `require`

Reply:
updated. `jest.config.json` was pointing directly at the typescript file instead of the transpile version.